lib/routes/routePUT.js

```diff
@@ -38,6 +39,8 @@ export default function routePUT(request, response, log, statsClient) {
         request.post = '';
         const post = [];
         let postLength = 0;
+        // Send continue message for streaming V4 requests, if applicable.
+        writeContinue(request, response);
```

Comment:
_Suggestion_: If I got it right - why not removing `writeContinue(...)` here and add `putBucket` and `putBucketACL` to the list `lib/utilities/writeContinue.js`? It might be clearer.